Station.json

```diff
@@ -10193,6 +10193,116 @@
 			"platforms": 2,
 			"laea": true
 		},
+		{
+			"group": 1,
+			"name": "Roermond",
+			"ril100": "XNRM",
+			"x": -67,
+			"y": 95,
+			"platformLength": 580,
+			"platforms": 3,
+			"laea": true
+		},
+		{
+			"group": 1,
+			"name": "Sittard",
+			"ril100": "XNSI",
+			"x": -85,
+			"y": 126,
+			"platformLength": 770,
+			"platforms": 4,
+			"laea": true
```

Comment:
Sind die Koordinaten hier tatsächlich entsprechend laea transformiert?

Reply:
Wenn das ETRS89 meint?

Reply:
Ähhhh... Ich glaube, ja, bin mir aber nicht ganz sicher 😅

Ich habe da mit der PROJ-Bibliothek die EPSG:3035-Projektion (die bei der Wikipedia-Karte dabeistand) genommen und dann die WSG84-Koordinaten von der Bibliothek mit der Projektion in x/y-Koordinaten übersetzen lassen und dann noch skaliert, dass es mit dem Koordinatensystem der Karte passt...

Reply:
Wobei ich einen kleinen Fehler dabei gemacht habe, da es bei den Koordinaten nicht vom WGS84 sondern GRS 1980 ausgegangen ist...
Das muss ich nachher mal korrigieren.